lesson07/assignment/score-more/src/main.js

```diff
@@ -0,0 +1,48 @@
+import Vue from 'vue'
+import { BootstrapVue, IconsPlugin } from 'bootstrap-vue'
```

Comment:
You don't have to use vue-bootstrap.  It is a little confusing, but vue bootstrap is just a vue component library that uses bootstrap.  You would need to use their documentation to refactor your code to use their components. 

To use vanilla bootstrap, you can comment out the vue-bootstrap lines and just add:

import 'bootstrap';

to your main.js file and your menu will work again.  Bootstrap and jquery were already added as dependencies in your package.json, so that is all you should need to do.

![image](https://user-images.githubusercontent.com/7476053/191305306-cd5e1279-b9fd-4d7f-a5e0-918b5dca84b9.png)


![image](https://user-images.githubusercontent.com/7476053/191305096-3d1a84ce-987e-44e8-af3e-bfd1ad7da787.png)

Reply:
Thank you so much for the help.  That got it all working again.

I was trying to resolve the extra whitespace below the footer.  You had mentioned this in a previous assignment, and you recommend try setting the min-height: 100vh on the body... etc.  I tried that and did some searches but none of the solutions seem to work.  I did some detailed inspection and discovered that the grid-area assignment is not being handled correctly, all of the content is ending up in the "Header" assignment and the whitespace are the Navbar, Content and Footer.

![image](https://user-images.githubusercontent.com/64674337/191414901-ffc3abe4-2e64-4c25-aa0f-8d6855b6cc02.png)

I could not figure out why the assignments are not being handled correctly.  You can even see that the footer style="grid-area: Footer / Footer  / Footer / Footer"

Reply:
I stumbled upon a fix that seems to work and makes sense to me.  I changed my scss body style to the #app, this then makes the grid assignments fall right into place.  The footer whitespace is now removed!

![image](https://user-images.githubusercontent.com/64674337/191419155-2778d2f0-1620-4492-afee-be908e2d7927.png)